Makefile

```diff
@@ -1,11 +1,11 @@
 dev:
-	docker compose -f ./docker/docker-compose.dev.yml up --remove-orphans || make dev-down
+	mkdir -p ./server/www && docker compose -f ./docker/docker-compose.dev.yml up --remove-orphans || make dev-down
```

Comment:
Perhaps this should use the WEB_ROOT_PATH variable that @jrasm91 pointed to? We'd need to update the server to use it as well

Reply:
If we do that, I wonder if it might be better to add the middleware only if the directory exists rather than auto creating the directory so that if someone has a WEB_ROOT_PATH they forgot about or something we don't go creating directories in their project. It probably doesn't matter either way though

Reply:
The `WEB_ROOT_PATH` env is intended to be used in production, not in a development environment: its value is the path where the `www` folder is stored inside the container, not the host path.

Reply:
I think that's a good idea. Only apply the middleware if the folder exists. And don't create it in any of these scripts for development.

Reply:
Should the folder be guaranteed to exist in the production build?

Reply:
Yes it always will or we won't have a webapp anymore 😂